pages/script.js

```diff
@@ -0,0 +1,444 @@
+/**
```

Comment:
What is the use case of this class?

Reply:
only UI techno-lightnings on the background of page, just nice looking and good reference to naming. If it's not appropriate page decoration, and don't have any use-  i'm waiting for the words - **cleanse it!**

Reply:
At least we need to make sure that this script file has appropriate license (Apache 2.0, MIT or BSD).

Reply:
@pychik still need to fix this. I would remove this script if not strictly required.

Reply:
script and links removed in bd17a0d7ccf73af3f273f94adfb0159421f99532